### DISCUSSION TITLE
Run asset_check only from Dagit

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#24049] [Q&A]
I would like to run asset check only from dagit UI . I don't want to run it for each materialization. 
Is it possible to skip these automatic checks  ?

**Comment by garethbrickman** [comment]
I don't think so. [According to the docs:](https://docs.dagster.io/concepts/assets/asset-checks/define-execute-asset-checks#executing-checks) 

>Materializing an asset from the UI will also execute any checks defined for that asset. To execute a check without materializing the asset, use the Checks tab of the Asset's details page.
